types/iron/index.d.ts

```diff
@@ -0,0 +1,111 @@
+// Type definitions for iron 5.0
+// Project: https://github.com/hueniverse/iron
+// Definitions by: Simon Schick <https://github.com/simonschick>
+//                 Rafael Souza Fijalkowski <https://github.com/rafaelsouzaf>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.4
+
+/// <reference types="node" />
+
+export interface ISealOptionsSub {
+
+    /**
+     * the size of the salt (random buffer used to ensure that two identical objects will generate a different encrypted result.
+     */
+    saltBits: number;
+
+    /**
+     * the algorithm used ('aes-256-cbc' for encryption and 'sha256' for integrity are the only two supported at this time).
+     */
+    algorithm: string;
```

Comment:
You can restrict the algorithm type here to what's in the docs.

Reply:
Oh yes, you are right. But now I have a question. In the doc we can see 2 algorithms: 
```
'aes-256-cbc' for encryption
'sha256' for integrity
```
But in the code we can see 3 algorithms:

```
exports.algorithms = {
    'aes-128-ctr': { keyBits: 128, ivBits: 128 },       // Requires node 0.10.x
    'aes-256-cbc': { keyBits: 256, ivBits: 128 },
    'sha256': { keyBits: 256 }
};
```
So, should I update considering 2 or 3 algorithm? Maybe @hueniverse can help us.

Reply:
You can use `keyof IAlgorithms` I guess.

Btw, I know I should have mentioned this earlier but I dropped the habit of prefixing interfaces with `I`, and iirc DT generally discourages it.

Reply:
LOL I know. The initial code had ```I``` so I kept the rule. But tomorrow I can rename everything without problem.

I need to study and practices more about keyof, mapped types and related. Tomorrow I will spend some time on it 👍